src/pages/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Categories() {
+  return (
+    <div>Categories</div>
```

Comment:
- [ ] `required`
- [ ] great job so far 👏 
- [ ] however, categories should display only a button with text `Check status`.
- [ ] please make this change.